python/tests/integration/connectors/memory/test_azure_cosmosdb_memory_store.py

```diff
@@ -16,6 +16,10 @@
 
 from semantic_kernel.memory.memory_record import MemoryRecord
 from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+from semantic_kernel.utils.settings import azure_cosmos_db_settings_from_dot_env
+
+# Load environment variables
+(cosmos_api, cosmos_connstr) = azure_cosmos_db_settings_from_dot_env()
```

Comment:
@aayush3011 this is still the culprit of the integration test failures. We're introducing an attempt to get the cosmos db settings from the environment variables, which hadn't existed before.

Reply:
I have fixed this now, let me know if there is still some blocker for the Integrations tests on your side.